lightly/api/api_workflow_datasources.py

```diff
@@ -1,23 +1,27 @@
-from typing import List
-from lightly.openapi_generated.swagger_client.models.datasource_raw_samples_data import (
-    DatasourceRawSamplesData,
-)
-from lightly.openapi_generated.swagger_client.models.datasource_raw_samples_data_row import (
-    DatasourceRawSamplesDataRow,
-)
-from lightly.openapi_generated.swagger_client.models.datasource_processed_until_timestamp_request import (
-    DatasourceProcessedUntilTimestampRequest,
-)
-
 import time
+from typing import List, Tuple
 
-def _convert_raw_data_to_list(data: List[DatasourceRawSamplesDataRow]):
-    return [(d.file_name, d.read_url) for d in data]
+from lightly.openapi_generated.swagger_client.models.datasource_processed_until_timestamp_request import DatasourceProcessedUntilTimestampRequest
+from lightly.openapi_generated.swagger_client.models.datasource_raw_samples_data import DatasourceRawSamplesData
 
 
 class _DatasourcesMixin:
 
-    def download_raw_samples(self, from_: int = 0, to: int = None):
+    def download_raw_samples(
+        self, from_: int = 0, to: int = None
+    ) -> List[Tuple[str, str]]:
+        """Downloads all filenames and read urls from the datasource between `from_` and `to`.
+
+        Samples which have timestamp == `from_` or timestamp == `to` will also be included.
+        
+        Args:
+            from_ (int): Unix timestamp from which on samples are downloaded.
```

Comment:
Don't include the type in the docstring. It is sufficient to have it in the declaration.